tests/test_dbapi.py

```diff
@@ -143,28 +138,23 @@ def test_Exceptions(self):
         self.assertEqual(self.cx.ProgrammingError, monetdbe.ProgrammingError)
         self.assertEqual(self.cx.NotSupportedError, monetdbe.NotSupportedError)
 
-    def test_InTransaction(self):
```

Comment:
also here why remove?

Reply:
It is still there but I moved it to `VariousConnectionTests` because there were some problems with the Connection object management.